store/datastore/ddl/mysql/ddl_gen.go

```diff
@@ -725,3 +729,11 @@ UPDATE repos SET repo_fallback='false'
 var updateTableSetRepoFallbackAgain = `
 UPDATE repos SET repo_fallback='false'
 `
+
+//
+// 025_add_builds_changed_files_column.sql
+//
+
+var addBuildsChangedfilesColumn = `
+ALTER TABLE builds ADD COLUMN changed_files TEXT
```

Comment:
The column should have a default value of `[]`

Otherwise existing builds are not read from the DB, as meddler can't parse empty strings to json array.

Reply:
the problem is that "TEXT" columns cannot have a default value: https://stackoverflow.com/questions/3466872/why-cant-a-text-column-have-a-default-value-in-mysql

Should I maybe run an update all in the migration?

Reply:
I updated the code like that.